frontend/src/pages/Classrooms/InviteStudents/index.tsx

```
@@ -0,0 +1,55 @@
+import Panel from "@/components/Panel";
+import Button from "@/components/Button";
+import CopyLink from "@/components/CopyLink";
+import { useState, useContext, useEffect } from "react";
+import { SelectedClassroomContext } from "@/contexts/selectedClassroom";
+import { useNavigate } from "react-router-dom";
+import { postClassroomToken } from "@/api/classrooms";
+
+
+import "../styles.css";
+
+const InviteStudents: React.FC = () => {
+const navigate = useNavigate();
+const { selectedClassroom } = useContext(SelectedClassroomContext);
+const [link, setLink] = useState<string>("");
+
+useEffect(() => {
+    const handleCreateToken = async () => {
+        if (!selectedClassroom) {
+            return;
+        }
+        await postClassroomToken(selectedClassroom.id, "TA")
+            .then((data: ITokenResponse) => {
+                const url = "http://localhost:3000/app/token/apply?token=" + data.token;
+                setLink(url);
+            })
+            .catch((_) => {
+                // do nothing
```

Leave a comment explaining why we are suppressing this error

---

added better comment to indicate this